src/mesh/test/Test_Mesh_Interface.hh

```diff
@@ -0,0 +1,238 @@
+//----------------------------------*-C++-*----------------------------------//
+/*!
+ * \file   mesh/test/Test_Mesh_Interface.cc
+ * \author Ryan Wollaeger <wollaeger@lanl.gov>
```

Comment:
If this file gets any larger, we probably should generate a `libmesh_test.so` file and teach the unit tests to link to the test library. Don't do this yet - but think about it for future work.

Reply:
This is a good point - I accidentally labeled the file as .cc in the header, possibly subconsciously preparing the test library!